cli.js

```diff
@@ -120,10 +130,10 @@ function generate(filepath, email) {
 
 async function init() {
 	const results = globby.sync([
-		'code_of_conduct.*',
-		'code-of-conduct.*',
-		'.github/code_of_conduct.*',
-		'.github/code-of-conduct.*'
+		path.join(directory, 'code_of_conduct.*'),
+		path.join(directory, 'code-of-conduct.*'),
+		path.join(directory, '.github', 'code_of_conduct.*'),
+		path.join(directory, '.github', 'code-of-conduct.*')
```

Comment:
Note comment https://github.com/sindresorhus/conduct/pull/14#discussion_r228756744, you need to use `path.posix.join()` here as globs cannot have backslashes (Windows).